package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "language-negotiator-lambda",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
Nitpick, but I'm not sure this change warrants a major version bump. Maybe bump to 0.2.0 or 0.1.1?